db/schema.rb

```diff
@@ -25,6 +25,15 @@
     t.integer  "user_id"
   end
 
+  create_table "reviews", force: true do |t|
+    t.integer  "rating",     null: false
+    t.text     "comment",    null: false
+    t.integer  "user_id",    null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.integer  "buddy_id",   null: false
+  end
```

Comment:
Add indexes for foreign keys so you can quickly find reviews for a buddy or from a user.